src/docs/style/illustrations.mdx

```diff
@@ -0,0 +1,73 @@
+import { MaterialDesignDescription, ImageGrid } from '../../app/components';
+import { PictureAsPdf as PDFIcon } from '@material-ui/icons';
+import LightThemeImage from '../../app/assets/design-patterns/illustrations/illustration-light-theme.svg';
+import DarkThemeImage from '../../app/assets/design-patterns/illustrations/illustration-dark-theme.svg';
+import BackgroundShapeImage from '../../app/assets/design-patterns/illustrations/illustration-background.png';
+
+<!-- KEYWORDS: design illustration graph onboarding -->
+
+# Illustrations
+
+Illustrations are design elements that should be used mostly for onboarding screens and [empty states](/patterns/empty-states). Do not use illustrations where you are supposed to use icons.
+
+When creating your own illustrations, you should follow our guidelines as explained below. Alternatively, you may also choose to acquire existing artworks from repositories like [Icons8](https://icons8.com/illustrations), [Adobe Stock](https://stock.adobe.com/vectors), or [Humaaans Figma plugin](https://www.figma.com/community/plugin/739503328703046360). We recommend you finalizing the illustration with [Adobe Illustrator](https://www.adobe.com/products/illustrator.html), as it provides more export options.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Onboarding'}
+        url={'https://material.io/design/communication/onboarding.html'}
+        description={`Material Design's guidelines on onboarding screens.`}
+    />
+    <MaterialDesignDescription
+        title={'Google Marketing Platform Visual Guidelines'}
+        url={
+            'https://designguidelines.withgoogle.com/ads-branding/assets/1hquKP2EcBlDaWyaLkoezqd8r9KPjRgnp/visual-guidelines-google-marketing-platform.pdf#'
+        }
+        description={`You are encouraged to follow their illustration guidelines.`}
+        avatar={<PDFIcon style={{ width: 40, height: 40 }} />}
+    />
+</React.Fragment>
+
+## Designing the Illustration
+
+### Perspective
+
+Your illustration should be either in a horizontal perspective (neither looking up nor looking down) or in parallel projection.
```

Comment:
Are these standard design/art terms (horizontal perspective and parallel projection)?

Reply:
parallel projection: yes

horizontal perspective: I looked it up everywhere but couldn't find a proper way to say it (that term is called "the plain perspective: in Chinese, but it makes no sense here, and I didn't find any translation to that)

Reply:
Maybe something like "facing the subject matter parallel to the ground (not viewing it from above or below)"